node/tests/SharedTests.ts

```diff
@@ -2,16 +2,13 @@
  * Copyright Valkey GLIDE Project Contributors - SPDX Identifier: Apache-2.0
  */
 
-// This file contains tests common for standalone and cluster clients, it covers API defined in
```

Comment:
Don't delete please

Reply:
@Yury-Fridlyand I copied and pasted the upstream SharedTest and pasted in my work for LMOVE. what is another way for me to do it?

Reply:
Upstream got changed since that. Copy-paste again or revert the entire file and re-apply your relevant changes (tests).